packages/schema/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@keyshade/schema",
   "version": "1.0.0",
-  "main": "dist/src/index.js",
+  "main": "dist/src/index.types.js",
```

Comment:
I'm not sure if this change would be a good thing to make. What are your thoughts?

Reply:
Since '.' import would be 'index.types.js'. I was able to import it in packages easily without moduleResolution.

Reply:
Should I include this or keep it just index.js? @rajdip-b

Reply:
Ah! okay then that works for me.